modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/RelatedObjectEntryOpenAPIContributor.java

```diff
@@ -199,6 +203,59 @@ private String _getContentType(
 		return dtoConverter.getContentType();
 	}
 
+	private Operation _getDeleteOperation(
+		ObjectDefinition objectDefinition,
+		ObjectRelationship objectRelationship,
+		SystemObjectDefinitionMetadata systemObjectDefinitionMetadata) {
+
+		String upperCaseFirstLetterObjectRelationshipName =
+			StringUtil.upperCaseFirstLetter(objectRelationship.getName());
+
+		DTOConverter<?, ?> dtoConverter = _dtoConverterRegistry.getDTOConverter(
+			systemObjectDefinitionMetadata.getModelClassName());
+
+		return new Operation() {
+			{
+				operationId(
+					"delete" + upperCaseFirstLetterObjectRelationshipName);
+				parameters(
+					Arrays.asList(
+						new Parameter() {
+							{
+								in("path");
+								name(
+									_getIdParameterName(
+										dtoConverter.getContentType()));
+								required(true);
+							}
+						},
+						new Parameter() {
+							{
+								in("path");
+								name(
+									_getIdParameterName(
+										objectDefinition.getShortName()));
+								required(true);
+							}
+						}));
+				responses(
+					new ApiResponses() {
+						{
+							setDefault(
+								new ApiResponse() {
+									{
+										getDefault();
```

Comment:
Just to inform, with this code we are generating the following response:
```yaml
      responses:
        default: {}
```

In other cases with the REST Builder, for the delete methods we are generating the following:
```yaml
      responses:
        default:
          content:
            application/json: {}
            application/xml: {}
```
What do we want to generate?

Reply:
Probably the second for consistency.